docs/source/rpc.rst

```diff
@@ -58,8 +58,7 @@ used for applications such as model parallel training. In short, applications
 may send and receive gradient recording tensors over RPC. In the forward pass,
 we record when gradient recording tensors are sent over RPC and during the
 backward pass we use this information to perform a distributed backward pass
-using RPC. For more details see the design doc
```

Comment:
You may want to rebase to avoid merge conflicts as this is different in the latest master (it was updated to refer to the note when that PR got merged)